{{cookiecutter.plugin_name}}/src/{{cookiecutter.module_name}}/_writer.py

```diff
@@ -16,7 +16,17 @@
 
 
 def write_single_image(path: str, data: Any, meta: dict) -> List[str]:
-    """Writes a single image layer"""
+    """Writes a single image layer.
+    
+    Parameters
+    ----------
+    path : str
+    data : The layer data
```

Comment:
`meta` should be also documented. As it contains all layer attributes different from the data.